wagon-provider-test/src/main/java/org/apache/maven/wagon/http/HttpWagonTestCase.java

```diff
@@ -936,7 +936,7 @@ protected void checkHandlerResult( List<HandlerRequestResponse> handlerRequestRe
         {
             for ( int i = 0; i < expectedResponseCodes.length; i++ )
             {
-                success &= ( expectedResponseCodes[i] == handlerRequestResponses.get( i ).responseCode );
+                success &= expectedResponseCodes[i] == handlerRequestResponses.get( i ).responseCode;
```

Comment:
This change isn't wrong, but doesn't seem related to isEmpty methods. What's happening?